pkg/pipeline/builder/video.go

```diff
@@ -259,17 +259,6 @@ func (b *VideoBin) buildSDKInput() error {
 
 	if b.conf.VideoDecoding {
 		b.bin.SetGetSrcPad(b.getSrcPad)
```

Comment:
Is this special handling not needed anymore?

Reply:
I'm not sure it ever was, we can just EOS the videoTestSrc bin as normal and skip the EOS on all the appSrcBins